CHANGELOG.md

```diff
@@ -5,10 +5,16 @@ Versioned according to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [0.8.3] - 2020-05-12
```

Comment:
I follow the convention to only change version numbers when releasing and adding changes under the `Unreleased` headline.

Reply:
Right, but this is a hen and egg problem, isn't it? Or do you make the change to the version number/changelog directly on master?

Reply:
Yeah, I make the version changes directly in master:

* Update changelog
* Update setup.py (or package.json, Makefile `VERSION` macro, `.version` or whatever the scheme)
* Commit as `:package: v0.0.0`
* git tag v0.0.0
* git push; git push --tags
* Release to PyPI, npm, nexus etc.
* Create a release on GitHub based on the tag.

For more complex setups (like core's mult-module setup), I have a little cheatsheet so I don't forget the steps.